Formula/couchbase-shell.rb

```diff
@@ -0,0 +1,34 @@
+class CouchbaseShell < Formula
+  desc "Modern and fun shell for Couchbase Server and Capella"
+  homepage "https://couchbase.sh"
+  url "https://github.com/couchbaselabs/couchbase-shell/archive/v0.75.0.tar.gz"
+  sha256 "0f653b03421b402bd6827b77823a84f9d7364c2872e23e4baa6cf04141625629"
+  license "Apache-2.0"
+  head "https://github.com/couchbaselabs/couchbase-shell.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+    regex(%r{href=.*?/tag/v?(\d+(?:[._]\d+)+)["' >]}i)
```

Comment:
I don't think this needs a specific regex right? What does it catch otherwise?

Reply:
The whole formula is based heavily on the nushell formula (https://github.com/Homebrew/homebrew-core/blob/72dbdc0d763701ab980027cc2424c062baeaf961/Formula/nushell.rb) as that's what couchbase-shell is built on top of. Happy to update this formula with your suggestions.